sddi/Dockerfile

```diff
@@ -1,107 +1,361 @@
+
+RUN set -ex && \
+    curl -o /wheels/ckanext-clamav-requirements.txt \
+        ${CKANEXT_CLAMAV_GITHUB_URL}/raw/${CKANEXT_CLAMAV_VERSION}/requirements.txt && \
+    pip install -r /wheels/ckanext-clamav-requirements.txt && \
+    pip wheel --wheel-dir=/wheels \
+        git+${CKANEXT_CLAMAV_GITHUB_URL}.git@${CKANEXT_CLAMAV_VERSION}#egg=ckanext-clamav
+
+# ckanext-dcat ##########################################################
+ARG CKANEXT_DCAT_VERSION="v1.5.1"
+ENV CKANEXT_DCAT_VERSION=${CKANEXT_DCAT_VERSION}
+ENV CKANEXT_DCAT_GITHUB_URL="https://github.com/ckan/ckanext-dcat"
+
+RUN set -ex && \
+    curl -o /wheels/ckanext-dcat-requirements.txt \
+        https://raw.githubusercontent.com/ckan/ckanext-dcat/${CKANEXT_DCAT_VERSION}/requirements.txt && \
+    pip install -r /wheels/ckanext-dcat-requirements.txt && \
+    pip wheel --wheel-dir=/wheels \
+        git+${CKANEXT_DCAT_GITHUB_URL}.git@${CKANEXT_DCAT_VERSION}#egg=ckanext-dcat
+
+# ckanext-security ######################################################
+ARG CKANEXT_SECURITY_VERSION="5724341"
+ENV CKANEXT_SECURITY_VERSION=${CKANEXT_SECURITY_VERSION}
+ENV CKANEXT_SECURITY_GITHUB_URL="https://github.com/MarijaKnezevic/ckanext-security"
     
-# #############################################################################
-# # Runtime stage
-# #############################################################################
-FROM ${BASEIMAGE_REPOSITORY}:${BASEIMAGE_VERSION} as runtime
+RUN set -ex && \
+    curl -o /wheels/ckanext-security-requirements.txt \
+        https://raw.githubusercontent.com/MarijaKnezevic/ckanext-security/${CKANEXT_SECURITY_VERSION}/requirements.txt && \
+    pip install -r /wheels/ckanext-security-requirements.txt && \
+    pip wheel --wheel-dir=/wheels \
+        git+${CKANEXT_SECURITY_GITHUB_URL}.git@${CKANEXT_SECURITY_VERSION}#egg=ckanext-security
+
+###############################################################################
+# Runtime stage
+###############################################################################
+FROM ckan/ckan-base:2.11.1
+
+ENV APP_DIR=/srv/app
+ENV SRC_DIR=/srv/app/src
+ENV CKAN_DIR=${SRC_DIR}/ckan
+ENV DATA_DIR=/srv/app/data
+ENV PIP_SRC=${SRC_DIR}
+ENV CKAN_INI=${APP_DIR}/production.ini
+ENV CKAN_STORAGE_PATH=/var/lib/ckan
+ENV UWSGI_HARAKIRI=50
 
 USER root
 
-ENV CKAN__PLUGINS "image_view text_view recline_view webpage_view datastore datapusher \
-  hierarchy_display hierarchy_form display_group relation \
-  spatial_metadata spatial_query datesearch repeating composite scheming_datasets \
-  password_policy resource_proxy geo_view geojson_view wmts_view shp_view \
-  dcat dcat_json_interface structured_data \
-  restricted clamav \
-  envvars"
+# Setting the locale
+ENV LC_ALL="en_US.UTF-8"
+RUN apt-get update && apt-get install --no-install-recommends -y locales
+RUN sed -i "/$LC_ALL/s/^# //g" /etc/locale.gen
+RUN dpkg-reconfigure --frontend=noninteractive locales
+RUN update-locale LANG=${LC_ALL}
```

Comment:
Is something similar done in the base image?

Reply:
The repeated lines are removed.